pages.pt_BR/common/mosquitto.md

```diff
@@ -0,0 +1,20 @@
+# mosquitto
+
+> Um broker de MQTT.
+> Mais informaçōes: <https://mosquitto.org/>.
+
+- Iniciar mosquitto:
+
+`mosquitto`
+
+- Especificar um arquivo de configuração para usar:
+
+`mosquitto --config-file {{caminho/para/arquivo/configuracao.conf}}`
+
+- Escutar em uma porta específica:
+
+`mosquitto --port {{8883}}`
+
+- Colocar como um processo rodando em background:
```

Comment:
I am not sure about this description. Is this correct?

> Note: I am not a native speaker, I checked this with Google Translate.

Reply:
Perhaps this better:
```suggestion
- Cria um processo rodando em background:
```